src/blockConfig.js

```diff
@@ -0,0 +1,12 @@
+export const configSchema = ElementPropTypes => {
+    return {
+        text: {
+            label: 'Text Content',
```

Comment:
We've updated all our labels to use sentence casing. It's not enforced or anything, but maybe provide that as the default.